src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { People } from '../People/PeopleTable';
+import { Person } from '../../types';
+
+type Props = {
+  people: Person[],
+  setPeople: (newValue: Person[]) => void,
+};
+
+export const PeoplePage: React.FC<Props> = ({ setPeople, people }) => {
+  const [loading, setLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+
+  useEffect(() => {
+    setHasError(false);
+    setLoading(true);
+
+    (async () => {
+      try {
+        setPeople(await getPeople());
+      } catch {
+        setHasError(true);
+      } finally {
+        setLoading(false);
+      }
+    })();
```

Comment:
As you get people here, you actually don't need to fetch them inside the App component